### DISCUSSION TITLE
How to stack item past 64

### DISCUSSION THREAD

**Discussion by okay1204** [stefvanschie/IF#224] [Q&A] [answered]
I'm trying to make an amount selector from 1-100, but this would require stacking an item past 64. It's theoretically possible as [this Inventory method](https://hub.spigotmc.org/javadocs/spigot/org/bukkit/inventory/Inventory.html#setMaxStackSize(int)) exists, Would it be possible to use this in something like a ChestGUI?

**Comment by stefvanschie** [comment, marked as answer]
No, this is not possible. From my testing, this method is pretty unstable and doesn't always display the stack size properly. You can grab the underlying inventory yourself and call this, just note that there's no guarantee that IF doesn't create a different inventory prior to showing the gui, so you may be calling that method on an inentory that won't be used in the end.

**Reply by okay1204** [reply]
I've already tried grabbing the inventory property from the ChestGUI using reflection and it was pretty buggy.. so yeah I guess it's not possible then. Thanks for answering though!
